content/css/jenkins.css

```diff
@@ -1289,13 +1302,13 @@ blockquote {
 
 .container.blog-post .events .ji-item-list>.event>a {
   text-align: left;
-  padding: .25rem 0 .25rem 5.5rem;
+  padding: .25rem 0 0 5.5rem;
   width: auto;
-  margin: .25rem 1rem .25rem .25rem;
-  border: 1px solid #ccc;
-  border-radius: 3px;
+  margin: .25rem .5rem .25rem .25rem;
+  /* border: 1px solid #ccc; */
+  /* border-radius: 3px; */
   background: #fff;
-  box-shadow: rgba(0, 0, 0, .15) 0 2px 2px;
+  /* box-shadow: rgba(0, 0, 0, .15) 0 2px 2px; */
```

Comment:
There is basically never a need to keep code but only comment it out, we have Git history for that.